src/bumerang/db/databasecreator.py

```diff
@@ -44,10 +44,13 @@ def _create_requests_table(self):
         query.create_table("""
             CREATE TABLE IF NOT EXISTS br_request(
                 ID SERIAL PRIMARY KEY,
+                USER_ID INT REFERENCES br_profile(id) NOT NULL,
```

Comment:
Since we're using a where statement on `USER_ID` an index would boost performance